app/javascript/css/uswds-settings.scss

```diff
@@ -0,0 +1,7 @@
+// Fixes required to find the correct folders
+// with webpacker 5.x
+$theme-font-path: "../../../fonts";
+$theme-image-path: "../../../img";
+$theme-hero-image: "../../../dist/img/hero.png";
+
+// Put your USWDS theme settings here
```

Comment:
Fun thing to add to test that it's working:

`$theme-color-base-ink: "red-50";`

Reply:
**praise:** I appreciate you including this note! it was an easy way for me to check locally, and think the comments make use of file clear